### PR TITLE
Fixed the search issue on homepage

### LINKED ISSUE #665: Image search from homepage won't update after a previous search
## Describe the bug
When we search from the homepage, and then return to it to perform a new search then the old results are shown .It's an extension of #616 
## To Reproduce
1. Go to 'https://search.creativecommons.org/'
2.Search something
3. Return to home page
4.Search something else
5. You'll see the previous results with new query in search box

## Expected behavior
The image results should be updated 

### Search something
image

### Press the back button to return to homepage
image

### Search Something else by clicking on search or return key
image

### You'll see the previous results
image
 

## Desktop (please complete the following information)
 - OS: Windows 10
 - BrowserChrome
 - Version 79.0.3945.130

## Additional context
This extends from #616

### PR DESCRIPTION
**Pull request by DantrazTrev**
Fixes #665 

Now the images for search are reloaded as soon as search query is updated from homepage

ezgif com-video-to-gif

---

Developer Certificate of Origin Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1 Letterman Drive Suite D4700 San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by akmadian** [comment]
I'm marking this as blocked so we can include a fix for #726, which is essentially the same issue.

**Comment by DantrazTrev** [comment]
I've done the changes however I remember that issue that there was a duplicate of #726

File: src/components/HeroSection.vue [hunk 1]
```diff
@@ -74,6 +74,7 @@ export default {
   methods: {
     onSubmit() {
       this.$store.commit(SET_QUERY, { query: { q: this.form.searchTerm }, shouldNavigate: true });
+      this.$store.commit(SET_IMAGES, { images: [] });
```
Comment (brenoferreira) [inline src/components/HeroSection.vue]: I think it's probably worth trying to set the `images: []` directly in the `SET_QUERY` mutation handler. You can try changing this and I'll make tests here to see if it doesn't cause any issues

Comment (DantrazTrev) [reply src/components/HeroSection.vue]: Sure, I'll give it a try

**Comment by DantrazTrev** [comment]
Hi @brenoferreira I'm think done with the requested changes , please tell me if this was what you requested and sorry for being late with it

**Comment by brenoferreira** [comment]
This was fixed in another pull request. But thanks for your contribution

**Pull request closed without merging**